modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ScriptPathAttributeGenerator.cs

```diff
@@ -14,11 +14,8 @@ public class ScriptPathAttributeGenerator : ISourceGenerator
     {
         public void Execute(GeneratorExecutionContext context)
         {
-            if (context.TryGetGlobalAnalyzerProperty("GodotScriptPathAttributeGenerator", out string? toggle)
-                && toggle == "disabled")
-            {
+            if (context.AreGodotSourceGeneratorsDisabled())
                 return;
-            }
 
             // NOTE: IsNullOrEmpty doesn't work well with nullable checks
```

Comment:
I don't really remember what this was about, but I think what it meant is that IDEs (at least Rider) still showed warnings about possible null dereferencing when using `IsNullOrEmpty` instead of `== null`. Not sure if this is still the case now.

Reply:
I use VSCode and null analysis works well with `string.IsNullOrEmpty` when using `netstandard2.1+`, I assume Rider uses Omnisharp too so it should have the same behavior.

I think your issues might have been because this project was targeting [`netstandard2.0`](https://github.com/godotengine/godot/blob/91b97dac03996c4b8791633ea6fa8fdc47852cb6/modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Godot.SourceGenerators.csproj#L3) before and the method was not annotated for nullability until `netstandard2.1`.

Reply:
> I think your issues might have been because this project was targeting netstandard2.0 before and the method was not annotated for nullability until netstandard2.1.

Yes, that's it. No longer can we not use the attribute in .NET Standard 2.0, but we don't get the nullable code analysis either. What was confusing me is that `Go to Declaration` in Rider was sending me to a newer version that was correctly annotated...

Reply:
I don't know if it's possible to target a newer version, or if it's recommended to stay with netstandard2.0 for compatibility with MSBuild and IDEs. For now, I prefer to be cautious and not change it.